### PR TITLE
Replaced xrange with range

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/frame/test_block_internals.py [hunk 1]
```diff
@@ -585,10 +585,6 @@ def test_strange_column_corruption_issue(self):
         df = DataFrame(index=[0, 1])
         df[0] = np.nan
         wasCol = {}
-        # uncommenting these makes the results match
-        # for col in xrange(100, 200):
```
Comment (MomIsBestFriend) [inline pandas/tests/frame/test_block_internals.py]: This code block is sitting commented over for over 4 years. thtat's the only reason I deleted it all. and not just changed the ```xrange``` to ```range```

**Review by topper-123 — APPROVED** [review]
Looks good. Any idea why  xrange even worked in the pyx file? I would presume ot should fail when we`re on a python3 code base.

**Comment by jbrockmendel** [comment]
LGTM

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Nice - thanks @MomIsBestFriend

**Pull request merged**